Building fails when selecting `ilp-cbc` feature

Kind of a Rust newbie; sorry if this is a basic misunderstanding!

Here's what I'm seeing when running `cargo build --features ilp-cbc`:

```

error[E0432]: unresolved imports `crate::generate_random_egraph`, `crate::ELABORATE_TESTING`
    --> src/extract/faster_ilp_cbc.rs:1157:34
     |
1157 |         faster_ilp_cbc::extract, generate_random_egraph, ELABORATE_TESTING, EPSILON_ALLOWANCE,
     |                                  ^^^^^^^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^ no `ELABORATE_TESTING` in the root
     |                                  |
     |                                  no `generate_random_egraph` in the root

error[E0107]: struct takes 0 generic arguments but 1 generic argument was supplied
   --> src/main.rs:98:53
    |
98  |                 extractor: extract::faster_ilp_cbc::FasterCbcExtractor::<10>.boxed(),
    |                                                     ^^^^^^^^^^^^^^^^^^------ help: remove these generics
    |                                                     |
    |                                                     expected 0 generic arguments
    |
note: struct defined here, with 0 generic parameters
   --> src/extract/faster_ilp_cbc.rs:184:12
    |
184 | pub struct FasterCbcExtractor;
    |            ^^^^^^^^^^^^^^^^^^

Some errors have detailed explanations: E0107, E0432.
For more information about an error, try `rustc --explain E0107`.
error: could not compile `extraction-gym` (bin "extraction-gym") due to 2 previous errors
```

--- a/src/extract/faster_ilp_cbc.rs
+++ b/src/extract/faster_ilp_cbc.rs
@@ -1151,11 +1151,12 @@ fn cycle_dfs(
     }
 }
 
+#[cfg(test)]
 mod test {
     use super::Config;
-    use crate::{
-        faster_ilp_cbc::extract, generate_random_egraph, ELABORATE_TESTING, EPSILON_ALLOWANCE,
-    };
+    use crate::test::{generate_random_egraph, ELABORATE_TESTING};
+
+    use crate::{faster_ilp_cbc::extract, EPSILON_ALLOWANCE};
     use rand::Rng;
     pub type Cost = ordered_float::NotNan<f64>;
 
@@ -1236,7 +1237,7 @@ mod test {
                 for _ in 0..CONFIGS_TO_TEST {
                     configs.push(generate_random_config());
                 }
-                test_configs(&configs, crate::test_save_path(stringify!($name)));
+                test_configs(&configs, crate::test::test_save_path(stringify!($name)));
             }
         )*
     }

--- a/src/main.rs
+++ b/src/main.rs
@@ -93,7 +93,7 @@ fn extractors() -> IndexMap<&'static str, ExtractorDetail> {
         (
             "faster-ilp-cbc",
             ExtractorDetail {
-                extractor: extract::faster_ilp_cbc::FasterCbcExtractor::<10>.boxed(),
+                extractor: extract::faster_ilp_cbc::FasterCbcExtractor.boxed(),
                 optimal: Optimal::DAG,
                 use_for_bench: true,
             },
@@ -170,4 +170,4 @@ fn main() {
 }
 
 #[cfg(test)]
-mod test;
+pub mod test;

